tests/new_api/test_device.py

```diff
@@ -324,13 +326,19 @@ def test_preprocess(self, adjoint):
 
     @pytest.mark.parametrize(
         "op, is_trainable",
-        [
-            (qml.StatePrep([1 / np.sqrt(2), 1 / np.sqrt(2)], wires=0), False),
-            (qml.StatePrep(qml.numpy.array([1 / np.sqrt(2), 1 / np.sqrt(2)]), wires=0), True),
-            (qml.StatePrep(np.array([1, 0]), wires=0), False),
-            (qml.BasisState([1, 1], wires=[0, 1]), False),
-            (qml.BasisState(qml.numpy.array([1, 1]), wires=[0, 1]), True),
-        ],
+        (
+            [
+                (qml.StatePrep([1 / np.sqrt(2), 1 / np.sqrt(2)], wires=0), False),
+                (qml.StatePrep(qml.numpy.array([1 / np.sqrt(2), 1 / np.sqrt(2)]), wires=0), True),
+                (qml.StatePrep(np.array([1, 0]), wires=0), False),
+                (qml.BasisState([1, 1], wires=[0, 1]), False),
+                (qml.BasisState(qml.numpy.array([1, 1]), wires=[0, 1]), True),
```

Comment:
We decompose them right now. right?

Reply:
Not really. Will work on the statevector to MPS decomposition after get all the measurement support